### ISSUE TITLE
Can't get pokemon to show up on the map

### ISSUE THREAD

**Issue by KouhaiKiller** [AHAAAAAAA/PokemonGo-Map#1084]
I'm using the dev branch with python 2.7 on Windows. When I run the server, it starts correctly without any syntax errors and begins to scan for pokemon. I made sure to install all the dependencies multiple times as well. However, when I go to the map (localhost:5000) no pokemon appear, just a blank map. Can anyone help me with this? I've had the same problem with the master branch too.

This is the command I use: python runserver.py -a ptc -u USERNAME-p PASSWORD -l "coordinates" -st 10

**Comment by Zach1993** [comment]
Just had the same problem. All the pokemon that show up are from a while ago and are all expired.

**Comment by KeithPetro** [comment]
What are you getting as the output of the server in commandline?

**Comment by KouhaiKiller** [comment]
4260/0.0000 (lat/lng/alt)
2016-07-21 20:09:49,673 [     search] [   INFO] Attempting login to Pokemon Go.
2016-07-21 20:09:49,676 [   auth_ptc] [   INFO] PTC login for: KouhaiKiller
2016-07-21 20:09:49,836 [  _internal] [   INFO]  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2016-07-21 20:09:50,266 [   auth_ptc] [   INFO] PTC Login successful
2016-07-21 20:09:50,651 [     search] [   INFO] Login to Pokemon Go successful.
2016-07-21 20:09:50,651 [     search] [   INFO] Scanning step 1 of 100.
2016-07-21 20:09:50,913 [     models] [   INFO] Upserting 1 pokemon
2016-07-21 20:09:51,000 [     models] [   INFO] Upserting 1 pokestops
2016-07-21 20:09:51,066 [     models] [   INFO] Upserting 1 gyms
2016-07-21 20:09:51,142 [     search] [   INFO] Completed  1.00% of scan.
2016-07-21 20:09:52,144 [     search] [   INFO] Scanning step 2 of 100.
2016-07-21 20:09:52,269 [     models] [   INFO] Upserting 2 pokemon
2016-07-21 20:09:52,411 [     models] [   INFO] Upserting 1 pokestops
2016-07-21 20:09:52,484 [     models] [   INFO] Upserting 1 gyms
2016-07-21 20:09:52,558 [     search] [   INFO] Completed  2.00% of scan.


...and so on. It scans, gets to 100%, then starts the process over again.

**Comment by gautamsawhney** [comment]
I also have the same problem, not getting anything on the map.

**Comment by rmad17** [comment]
Same here

**Comment by gautamsawhney** [comment]
I think it's not the issue with the app,  in India normally opening the app I can't see any pokestops, gyms or pokemons.

**Comment by ottocycle** [comment]
Yes. App is working fine for other countries. Niantic might have disabled the game all together for india to reduce server load.

**Comment by KouhaiKiller** [comment]
Well I live in the US, just for reference. There shouldn't be any problem here, and it's not like the servers are down either. Also, I made sure to port forward port 5000, so that's not the issue either.

**Comment by Jshak** [comment]
Dude having the same problem here in NEPAL......MAp shows pokestops but no pokemons :lollipop:

**Comment by BillyWM** [comment]
Which browser? This problem happens with IE11 (due to an issue soon to be fixed).

**Comment by GowthamRayar** [comment]
Yep I too facing same problem in Bangalore right from i woke up

**Comment by KouhaiKiller** [comment]
I'm using Chrome. I figured out part of the issue, it seems there were still old python processes running in the background, so I closed them. However, now when I run the server again, the GUI displays when I go to localhost:5000 (asking me whether to show gyms, pokemon, etc.) and there's not even a map OR pokemon.
A small update though: I got it to work on the master build now, so this is only a develop build issue.
